### PR TITLE
New documentation about HSTS enablement

### PR DESCRIPTION
**Pull request by renner**
# Description

This is a new paragraph to add documentation on how to enable HSTS headers that we are shipping disabled by default.

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Belongs to https://github.com/uyuni-project/uyuni/pull/5302.

### CODE REVIEW

**Review by thesp0nge — APPROVED** [review]

File: modules/administration/pages/ssl-certs-hsts.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[ssl-certs-hsts]]
+= HTTP Strict Transport Security
+
+For additional security with regard to certificates and better protection against specific MITM attacks, it is possible to enable [HTTP Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security).
+This is done by uncommenting the following line in the Apache Webserver configuration on the {productname} server as well as {productname} proxies if used:
```
Comment (jcayouette) [inline modules/administration/pages/ssl-certs-hsts.adoc:5]: To enable Strict Transport Security uncomment the following line in the Apache Webserver configuration on the {productname} server and {productname} proxies.

File: modules/administration/pages/ssl-certs-hsts.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+----
+# Enable HSTS
+Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
+----
+
+Be aware that once HSTS is enabled, browsers will no longer allow users to make exceptions when trusting invalid or self-signed certificates, so in order to be able to access {productname}, the respective CA certificate will need to be imported to browsers of all the users.
```
Comment (jcayouette) [inline modules/administration/pages/ssl-certs-hsts.adoc:12]: browsers will no longer allow users to make exceptions when trusting invalid or self-signed certificates. To ensure access to the {productname} server, the respective CA certificate will need to be imported to browsers of all users.

**Comment by renner** [comment]
Thanks Joseph. The text has changed a bit using the release notes as input, could you please give it another review now?

**Review by mcalmer — APPROVED** [review]

**Review by juliogonzalez — APPROVED** [review]
LGTM

**Review by keichwa — COMMENTED** [review]
Later we (doc-squad) can normalize it a little bit more according to the documentation style guide.

**Pull request merged**